frontend/src/components/createProjectForm.tsx

```diff
@@ -0,0 +1,367 @@
+                }}
+                onSubmit={handleSubmit}
+            >
+                <Form>
+                    <h2>{capitalize(t("create project"))}</h2>
+                    <Field
+                        className="form-control mb-2"
+                        label={capitalize(t("project name"))}
+                        name="name"
+                        data-testid="projectname-input"
+                        placeholder={capitalize(t("project name placeholder"))}
+                        required
+                    />
+                    <Field
+                        className="form-control mb-2"
+                        label={capitalize(t("enter project info"))}
+                        name="info"
+                        data-testid="projectinfo-input"
+                        placeholder={capitalize(t("project info placeholder"))}
+                    />
+                    {goals.map((goal: string, index: number) => (
+                        <Row key={index}>
+                            <Col>{goal}</Col>
+                            <Col xs={1}>
+                                <a>
+                                    <Image
+                                        onClick={() => handleDeleteGoal(index)}
+                                        alt=""
+                                        src={"/resources/delete.svg"}
+                                        width="15"
+                                        height="15"
+                                    />
+                                </a>
+                            </Col>
+                        </Row>
+                    ))}
+                    <Field
+                        className="form-control mb-2"
+                        label={capitalize(t("enter project goal"))}
+                        data-testid="goal-input"
+                        value={goalInput}
+                        placeholder={capitalize(t("project goal placeholder"))}
+                        onChange={handleChangeGoalInput}
+                    />
+                    <button
+                        className="btn btn-secondary"
+                        type="button"
+                        onClick={handleAddGoal}
+                        data-testid="add-goal-button"
+                    >
+                        {capitalize(t("add goal"))}
+                    </button>
+                    <Field
+                        className="form-control mb-2"
+                        label={capitalize(t("version control URL"))}
+                        name="versionManagement"
+                        data-testid="versionmanagement-input"
+                        placeholder={capitalize(t("version control placeholder"))}
+                        required
+                    />
+                    {coaches.map((coach: string, index: number) => (
+                        <Row key={index}>
+                            <Col>{coach}</Col>
```

Comment:
Not sure, is this a hard coded string?

Reply:
No, it's just every selected goal being mapped to an element that can be displayed.